cli/internal/specs/v0/spec_reader.go

```diff
@@ -177,10 +190,12 @@ func (r *SpecReader) GetDestinationNamesForSource(name string) []string {
 
 func NewSpecReader(paths []string) (*SpecReader, error) {
 	reader := &SpecReader{
-		sourcesMap:      make(map[string]*Source),
-		destinationsMap: make(map[string]*Destination),
-		Sources:         make([]*Source, 0),
-		Destinations:    make([]*Destination, 0),
+		sourcesMap:             make(map[string]*Source),
+		destinationsMap:        make(map[string]*Destination),
+		Sources:                make([]*Source, 0),
```

Comment:
No need to `make` the slices.

Reply:
It was like this before, but we can clean it up in the next iteration